recipes/xoshiro-cpp/all/test_package/conanfile.py

```diff
@@ -0,0 +1,18 @@
+import os
+
+from conans import ConanFile, CMake, tools
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake", "cmake_find_package_multi"
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def test(self):
+        if not tools.cross_building(self.settings):
```

Comment:
```suggestion
        if not tools.cross_building(self):
```
Please use `cross_building` with `self`.

Please refer following:
https://github.com/conan-io/conan-center-index/blob/master/docs/package/all/test_package/conanfile.py